pkgs/by-name/co/cosmic-icons/package.nix

```
@@ -1,20 +1,20 @@
 {
-  lib,
-  stdenvNoCC,
   fetchFromGitHub,
+  hicolor-icon-theme,
   just,
+  lib,
   pop-icon-theme,
-  hicolor-icon-theme,
-  nix-update-script,
+  stdenvNoCC,
 }:
+
 stdenvNoCC.mkDerivation {
   pname = "cosmic-icons";
-  version = "1.0.0-alpha.3";
+  version = "1.0.0-alpha.4";
 
   src = fetchFromGitHub {
     owner = "pop-os";
     repo = "cosmic-icons";
-    rev = "epoch-1.0.0-alpha.3";
+    rev = "refs/tags/epoch-1.0.0-alpha.4";
```

```suggestion
    tag = "epoch-1.0.0-alpha.4";
```
Ditto for every other package.